test/integration/webhook/v1alpha1/workload_test.go

```diff
@@ -70,3 +70,76 @@ var _ = ginkgo.Describe("Workload defaulting webhook", func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("Workload validating webhook", func() {
+	ginkgo.Context("When creating a Workload", func() {
+		ginkgo.It("Should validate spec.podSets.count", func() {
```

Comment:
Should we just have a single object with a bunch of errors?

As stated above, we can have most of the coverage in unit tests.